src/main/java/net/minecraftforge/client/event/RenderLevelStageEvent.java

```diff
@@ -0,0 +1,145 @@
+         * Called regardless of if they sky actually renders or not.
+         */
+        public static final Stage AFTER_SKY = register("after_sky", null);
+        /**
+         * Use this to render custom block-like geometry into the world.
+         */
+        public static final Stage AFTER_SOLID_BLOCKS = register("after_solid_blocks", RenderType.solid());
+        /**
+         * Use this to render custom block-like geometry into the world.
+         */
+        public static final Stage AFTER_CUTOUT_MIPPED_BLOCKS_BLOCKS = register("after_cutout_mipped_blocks", RenderType.cutoutMipped());
+        /**
+         * Use this to render custom block-like geometry into the world.
+         */
+        public static final Stage AFTER_CUTOUT_BLOCKS = register("after_cutout_blocks", RenderType.cutout());
+        /**
+         * Use this to render custom block-like geometry into the world.
+         * Called within a fabulous graphics target. 
+         * Happens after block entities render. 
+         */
+        public static final Stage AFTER_TRANSLUCENT_BLOCKS = register("after_translucent_blocks", RenderType.translucent());
+        /**
+         * Use this to render custom block-like geometry into the world.
+         */
+        public static final Stage AFTER_TRIPWIRE_BLOCKS = register("after_tripwire_blocks", RenderType.tripwire());
+        /**
+         * Use this to render custom effects into the world, such as custom entity-like objects or special rendering effects.
+         * Called within a fabulous graphics target.
+         * Happens after entities render.
+         */
+        public static final Stage AFTER_PARTICLES = register("after_particles", null);
+        /**
+         * Use this to render custom weather effects into the world.
+         * Called within a fabulous graphics target.
+         */
+        public static final Stage AFTER_WEATHER = register("after_weather", null);
+
+        private final String name;
+
+        private Stage(String name)
+        {
+            this.name = name;
+        }
+
+        private static Stage register(ResourceLocation name, @Nullable RenderType renderType)
+        {
+            Stage stage = new Stage(name.toString());
+            if (renderType != null && RENDER_TYPE_STAGES.put(renderType, stage) != null)
+                throw new IllegalArgumentException("Attempted to replace an existing RenderLevelStageEvent.Stage for a RenderType: Stage = " + stage + ", RenderType = " + renderType);
+            return stage;
+        }
+
+        private static Stage register(String name, @Nullable RenderType renderType)
+        {
+            return register(new ResourceLocation("forge", name), renderType);
+        }
+
+        @Override
+        public String toString()
+        {
+            return this.name;
+        }
+
+        /**
+         * @return The Stage bound to the RenderType passed or null if no value is present.
```

Comment:
We can use the [`{@return}` inline javadoc tag](https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html#return) here:
```suggestion
         * {@return the Stage bound to the RenderType, or null if no value is present}
```